src/index.html

```diff
@@ -17,3 +17,32 @@ <h1 class="logo">MA</h1>
     <script src="scripts/main.js"></script>
   </body>
 </html>
+<script>
+  const logo = document.querySelector('.logo');
+
+  const promise1 = new Promise((resolve) => {
+    logo.addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
```

Comment:
Consider handling the case where the promise may never resolve, for example, if the user never clicks the logo. You might want to add a timeout or another condition to ensure that the promise doesn't hang indefinitely.